_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+`-w`, `--width` | Window width in pixels for the report. | Default is `1680`.| N/A
+`-l`, `--height` | Minimum window height in pixels for the report. | Default is `600`. | N/A
+`-n`, `--filename` | File name of the report. | Default is `reporting`. | FILENAME
+`-e`, `--transport` | Transport for sending the email. | For Amazon SES specify `ses`. SES requires an AWS config on your system to store the credentials. For SMTP use `smtp` and also specify the login credentials with `--smtpusername` and `--smtppassword`. | TRANSPORT
+`-s`, `--from` | The email address of the sender. | For example, `user@amazon.com`. | FROM
+`-r`, `--to` | The email address of the recipient. | For example, `user@amazon.com`. | TO
+`--smtphost` | The hostname of the SMTP server. | For example, `SMTP_HOST`. | SMTP_HOST
+`--smtpport` | The port for connection. | For example, `SMTP_PORT`. | SMTP_PORT
+`--smtpsecure` | Use TLS when connecting to server. | For example, `SMTP_SECURE`. | SMTP_SECURE
+`--smtpusername` | The SMTP username.| For example, `SMTP_USERNAME`. | SMTP_USERNAME
+`--smtppassword` | The SMTP password.| For example, `SMTP_PASSWORD`. | SMTP_PASSWORD
+`--subject` | Email Subject text encased in quotes. | Can be any string. The default is "This is an email containing your dashboard report". | SUBJECT
+`-h`, `--help` | Display list of optional arguments from the command line. | N/A
+
+
+
+## Use environment variables with Reporting CLI
+
+Instead of explicitly providing values in the command line, you can save them as environment variables. The Reporting CLI reads environment variables from .env file inside the project.
+
+```
+.
+Reporting
+├── src
+│   └── .env
+```
+
+Each line should have the format NAME=VALUE.
+Each line that starts with a hashtag (#) is considered as a comment.
+Quotation marks (") don't get any special handling.
+
+Values from the command line argument have higher priority than the environment file. For example, if you add filename as *test* in *.env* file and also add `--filename report` command option, the generated report's name will be *report*.
+{: .note }
+
+### Example: Request a PNG report with environment variables set
+
+The following command requests a dashboard report with basic authentication in png format.
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --format png --auth basic --credentials admin:admin
+```
+
+Upon success, the report will download to the current directory.
+
+### Use Amazon SES to request an email with a report attachment
+
+To use Amazon SES as the email transport mechanism, the following prerequisites apply:
+
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily. The sender's email address must be verified on [Amazon SES](https://aws.amazon.com/ses/).
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily.
```

Comment:
This seems repeated few times `If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily.` Not sure if it's intentional. Also it's not relevant to SES.